src/CoreFx.Private.TestUtilities/src/System/IO/FileCleanupTestBase.cs

```diff
@@ -84,7 +85,37 @@ protected virtual void Dispose(bool disposing)
         /// <param name="index">An optional index value to use as a suffix on the file name.  Typically a loop index.</param>
         /// <param name="memberName">The member name of the function calling this method.</param>
         /// <param name="lineNumber">The line number of the function calling this method.</param>
-        protected string GetTestFileName(int? index = null, [CallerMemberName] string memberName = null, [CallerLineNumber] int lineNumber = 0) =>
+        protected string GetTestFileName(int? index = null, [CallerMemberName] string memberName = null, [CallerLineNumber] int lineNumber = 0)
+        {
+            string testFileName = GenerateTestFileName(index, memberName, lineNumber);
+            string testFilePath = Path.Combine(TestDirectory, testFileName);
+
+            int excessLength = testFilePath.Length - 255;
```

Comment:
I see this in two places, here and in the Debug.Assert, should it be a constant instead?

Reply:
Done